src/thriveopedia/pages/ThriveopediaMuseumPage.cs

```diff
@@ -26,14 +27,23 @@ public class ThriveopediaMuseumPage : ThriveopediaPage
     [Export]
     public NodePath LeaveGameConfirmationDialogPath = null!;
 
+    [Export]
+    public NodePath DeleteConfirmationDialogPath = null!;
+
+    [Export]
+    public NodePath DeletionFailedDialogPath = null!;
+
     private HFlowContainer cardContainer = null!;
     private Control welcomeLabel = null!;
     private VBoxContainer speciesPreviewContainer = null!;
     private SpeciesPreview speciesPreview = null!;
     private CellHexesPreview hexesPreview = null!;
     private CustomRichTextLabel speciesDetailsLabel = null!;
     private CustomConfirmationDialog leaveGameConfirmationDialog = null!;
+    private CustomConfirmationDialog deleteConfirmationDialog = null!;
+    private CustomConfirmationDialog deletionFailedDialog = null!;
     private PackedScene museumCardScene = null!;
+    private MuseumCard? cardToBeDeleted;
```

Comment:
```suggestion

    private MuseumCard? cardToBeDeleted;
```

Should add a blank line between blocks of logically different variables (Godot vs non-Godot components).